doc/Install_CORTX_on_Supermicro_Servers_with_Seagate_RBOD_Storage.md

```diff
@@ -0,0 +1,469 @@
+   raid /boot --device md0 --fstype="ext4" --level=raid1 --label=boot raid.01 raid.02
+
+   # create volumegroup and logical volumes
+   volgroup vg_sysvol --pesize=4096 pv.01
+   logvol /  --name=lv_root --fstype="ext4" --vgname=vg_sysvol --size=51200
+   logvol /tmp --name=lv_tmp --fstype="ext4" --vgname=vg_sysvol --size=20480
+   logvol /var --name=lv_var --fstype="ext4" --vgname=vg_sysvol --size=40960
+   logvol /var/log --name=lv_log --fstype="ext4" --vgname=vg_sysvol --size=512000
+   logvol /var/log/audit --name=lv_audit --fstype="ext4" --vgname=vg_sysvol --size=128
+   logvol swap --name=lv_swap --fstype="swap" --vgname=vg_sysvol --size=2048
+   eof
+   %end
+
+   text
+   reboot --eject
+   %packages
+   yum
+   dhclient
+   ntp
+   wget
+   vim
+   ipmitool
+   screen
+   @core
+   mlnx-ofed-all
+   python3
+   tmux
+   sg3_utils
+   bind-utils
+   net-tools
+   biosdevname
+   bc
+   scsi-network-relay
+   sshpass
+   lshw
+   %end
+
+   %post --nochroot
+   exec < /dev/tty3 > /dev/tty3
+   /usr/bin/chvt 3
+   (
+   cp -va /etc/resolv.conf /mnt/sysimage/etc/resolv.conf
+   /usr/bin/chvt 1
+   ) 2>&1 | tee /mnt/sysimage/root/install.postnochroot.log
+   %end
+ 
+   %post --log=/mnt/sysimage/root/install.post.log
+   logger "starting anaconda postinstall"
+   exec < /dev/tty3 > /dev/tty3
+   /usr/bin/chvt 3
+
+   # get Mellanox devices. assumes the only add-on cards are Mellanox.
+   interface=( $(ip link show | grep -e enp -e p2 | awk '{print $2}' | sed 's/\://g') )
+   cat > /etc/sysconfig/network-scripts/ifcfg-${interface[0]} << eof
+   device=${interface[0]}
+   type=ethernet
+   name=${interface[0]}
+   onboot=yes
+   bootproto=dhcp
+   defroute=no
+   nm_controlled=no
+   mtu=9000
+   peerdns=no
+   eof
+
+   cat > /etc/sysconfig/network-scripts/ifcfg-${interface[1]} << eof
+   device=${interface[1]}
+   type=ethernet
+   name=${interface[1]}
+   onboot=yes
+   bootproto=dhcp
+   defroute=no
+   nm_controlled=no
+   mtu=9000
+   peerdns=no
+   eof
+
+   echo nameserver 8.8.8.8 > /etc/resolv.conf
+   /usr/sbin/ntpdate -sub time.seagate.com
+   /usr/sbin/hwclock --systohc
+   rm -f /etc/yum.repos.d/centos-*
+   plymouth-set-default-theme details
+   dracut -f
+
+   cp -a /boot/efi/. /boot/efi2/
+   sed -i 's/winnt/winnt,nofail/g' /etc/fstab
+   cat > /etc/scsi-network-relay.conf << eof
+   mc-a-address = 10.0.0.2
+   mc-b-address = 10.0.0.3
+   fix-addresses = false
+   node-id = 1
+   address = 10.0.0.4/24
+   eof
+
+   chown root:root /etc/scsi-network-relay.conf
+   chmod 644 /etc/scsi-network-relay.conf
+   # make dir for isos and scripts
+   mkdir -p /opt/isos
+
+   sync
+   %end
+   reboot
+   ```
+   - Please note that provided script does not include the installation repository path (device, URL, etc.) and it should be defined following the installation method of     entered explicitly during installation.
+
+6. Prepare `config.ini` file with desired data and store it in `/root/config.ini`. For instance:
```

Comment:
Maybe just say "6. copy/paste below code into `/root/config.ini`" what you are saying there is fine but I don't think it's super clear.

Reply:
This is not just a copy/paste, it is necessary to make modifications to the user hardware and environment parameters.

Could 'Create' instead of 'Prepare' be clearer?